Source/Atomic/Resource/ResourceEvents.h

```diff
@@ -95,6 +95,11 @@ EVENT(E_PROJECTUNLOADEDNOTIFICATION, ProjecUnloadedNotification)
 {
 }
     
+/// User Preferences Changed
+EVENT(E_USERPREFERENCESCHANGEDNOTIFICAITON, UserPreferencesChangedNotification)

```

Comment:
Does this strictly need to be added as a native event?  Event types can be defined in script... I'll have a look at the handler and see about some thoughts there.